packages/core/src/extractors/types.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseNode, MetadataMode, TextNode } from "../Node";
+
+const DEFAULT_NODE_TEXT_TEMPLATE =
+  "\
+[Excerpt from document]\n{metadata_str}\n\
```

Comment:
This will need to be a function/SimplePrompt/JS template literal to work properly. LangchainJS does support the conversion of Python f-strings into JS prompts, but I don't feel like the juice is worth the squeeze. If folks prefer to use something other than JS template literals, there's a wide variety of templating options in the JS ecosystem from handlebars to ejs that people can choose.

Reply:
I ended up letting it go, thanks for pointing it out